sycl/test-e2e/PrivateAlloca/ValidUsage/private_alloca_bool_size.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %{build} -w -o %t.out
-// RUN: echo 1  | %{run} %t.out
+// RUN: echo 1 1  | %{run} %t.out
```

Comment:
In which cases is this needed? Aren't we reading a single number from stdin? Same for other tests.